src/PrestaShopBundle/Bridge/AdminController/Field/Field.php

```diff
@@ -105,9 +105,13 @@ protected function configureOptions(OptionsResolver $resolver): void
             'position',
             'search',
             'width',
+            'type',
+            'havingFilter',
```

Comment:
Is this one really needed since we have the `filter_key` already?
I'm not to understand exactly the purpose of this option

Reply:
i don't know the purpose of all options, but legacy handles it this way, so without too much thinking I added it as HelperList thrown error. But you are right, I will check and make sure its not some useless duplicate and check if filter_key covers it.

Reply:
apparently HavingFilter is related to adding HAVING clause to sql. Not gonna go deeper into it, but it seems to be important in legacy.